docs/man/git-lfs-faq.adoc

```diff
@@ -0,0 +1,144 @@
+problems.
+
+How do I track files that are already in a repository?::
+  If you want to track files that already exist in a repository, you need to do
+  two things.  First, you need to use `git lfs track` (or a manual modification
+  of `.gitattributes`) to mark the files as LFS files.  Then, you need to run
+  `git add --renormalize .` and commit the changes to the repository.
++
+If you skip this second step, then you'll end up with files that are marked as
+LFS files but are stored as Git files, which can lead to files which are always
+modified, as outlined in the FAQ entry above.  Note also that this doesn't
+change large files in your history.  To do that, use `git lfs migrate import
+--everything` instead, as specified in one of the entries below.
+
+How do I enable git diff to work on LFS files?::
+  You can run `git config diff.lfs.textconv cat`, which will produce normal
+  diffs if your files are text files.
+
+How do I enable git diff to work on LFS files based on extension or path?::
+  If the above solution is too broad, each entry in the `.gitattributes` file
+  can be customized by creating a custom global converter:
++
+[source,shell-session]
+----
+$ git config --global diff.lfstext.textconv cat
+----
++
+Any given `.gitattributes` entry for large text files can be customized to use
+  this global text converter (e.g., patch files), whereas binary formats can
+  continue to use the conventional lfs diff tool, like so:
++
+[source,shell-session]
+----
+$ cat .gitattributes
+....
+*.bin filter=lfs diff=lfs merge=lfs -text
+*.patch filter=lfs diff=lfstext merge=lfs -text
+....
+----
++
+Be advised that all developers sharing this repo with such a modified
+`.gitattributes` file must have similarly configured the `lfstext` text
+converter, whether globally or on per repository basis.
+
+How do I convert from using Git LFS to a plain Git repository?::
+  If you'd like to stop using Git LFS and switch back to storing your large
+  files in the plain Git format, you can do so with `git lfs migrate export
+  --everything`.  Note that you will need to provide and appropriate `--include`
+  option to match all the patterns that you currently have tracked in any ref.
++
+This also rewrites history, so the Git object IDs of many, if not all, of your
+objects will change.
+
+I'm using Git LFS, but I still see GitHub's large file error.  How do I fix this?::
+  GitHub rejects large files anywhere in the history of your repository, not
+  just in the latest commit.  If you're still seeing this message, then you have
+  some large files somewhere in your history, even if in the latest commits
+  you've moved them to Git LFS.
++
+To fix this, you can use `git lfs migrate import --everything` with an
+appropriate `--include` argument.  For example, if you wanted to move your
+`.jpg` and `.png` files into Git LFS, you can do that with
+`git lfs migrate import --everything --include="\*.jpg,*.png"`.  More complicated
```

Comment:
Just out of interest, is there a difference here between `\*` and plain `*`?

Reply:
This is required to make AsciiDoc not consider this bold.  Very fortunately, it's also syntactically correct according to the shell.